### ISSUE TITLE
Add banner, server description and tags for servers in the 'Server Browser'

### ISSUE THREAD

**Issue by PlatinMTA** [multitheftauto/mtasa-blue#2104]
**Is your feature request related to a problem? Please describe.**
It's kinda annoying having to add tags to your server name and kinda ruins the experience of searching new servers. From the sv browser, almost every server looks the same so you kinda need to go jumping from server to server in order to find something good and remarkable, or even something of your liking. Servers that say "RP" maybe are not really RP. Freeroam servers maybe are just default freeroams. Race servers maybe have DD and you don't really like DD and you could keep piling up.

#759 is related.

**Describe the solution you'd like**
A tag system (like implemented in the GoldSrc and Source games) could help reduce the issue of long names with a lot of tags for the server, giving much needed clarity in the server browser.

The server description would also help new users know more about the server they are about to join. Server owners could explain they intentions with the server more, give links to helpful websites or community rules and so on.

Lastly a banner would be awesome to have in order to display your community logo. This image should be fetch from a website outside of MTA's control, so maybe it's not worth it.

Also a languague option could help users find international communities or local ones.

All of this information could appear on the top of the player list.

![image](https://user-images.githubusercontent.com/23291573/109756169-2934cf80-7bc6-11eb-88bc-5ab470616329.png)

**Describe alternatives you've considered**
The banner and description are pretty much optional and not really needed to help improve the browsing server experience, but a languague and tag option is actually needed.

**Comment by Disinterpreter** [comment]
Examples from RAGE MP, another MP.
![image](https://user-images.githubusercontent.com/3816155/109765358-c9f6af80-7c05-11eb-8c2c-973d12b92715.png)
![image](https://user-images.githubusercontent.com/3816155/109765476-f6aac700-7c05-11eb-8b5d-39be3f333ca1.png)

**Comment by turret001** [comment]
yes please, I've been wanting to suggest this for a while glad someone took their time to do it.

**Comment by thalentdev** [comment]
> yes please, I've been wanting to suggest this for a while glad someone took their time to do it.

I'm happy for you.

![turret](https://user-images.githubusercontent.com/80006505/110216661-5c76a780-7ec1-11eb-9bef-82e9fc221173.png)

**Comment by Fernando-A-Rocha** [comment]
yes please. not only would it give MTA server browser a more modern feel, tags, languages and description is very useful and impacting

**Comment by Fernando-A-Rocha** [comment]
These features can be achieved with https://wiki.multitheftauto.com/wiki/Server_Browser_Rules
All that needs to be modified is the Server Browser user interface
